lang/cs/Org.Apache.REEF.Wake/Time/Runtime/RuntimeClock.cs

```diff
@@ -73,6 +74,11 @@ public sealed class RuntimeClock : IClock
             _runtimeStartHandler = runtimeStartHandler;
             _runtimeStopHandler = runtimeStopHandler;
             _idleHandler = idleHandler;
+            ++NumberOfInstantiations;
```

Comment:
I believe this was done for debugging....Do we need to keep it or remove it. In case we are keeping it, should we use Logger.Log for logging?